config/config.yaml

```diff
@@ -2,6 +2,7 @@ defaults:
   - model: wav2vec2
   - datasets:
     - common_voice_13_da
+    - coral_test_set
```

Comment:
I would leave this out for now. Instead, I think it would make sense to specify the training datasets and the test sets separately, rather than mixed up like it is currently. Let's leave that for a separate PR.

```suggestion
```

Reply:
Would still remove this, otherwise you'd get errors here currently.